src/reducers/index.js

```diff
@@ -1,11 +1,17 @@
-import {combineReducers} from "redux";
+import { combineReducers } from "redux";
```

Comment:
I think we got mixed up with the different folders. The reducers need to go in the client/src/reducers/index.js